app/main.py

```diff
@@ -1 +1,16 @@
-# write your code here
+def copy_file(command: str) -> None:
+
+    # Split command line into three parts
+    cmd, file_name, copy_file_name = command.split()
```

Comment:
If command.split() will have 2 or 4 elements your code will broke
Validate this